### ISSUE TITLE
Selected State does not annonce when ProgressBarAndroid Component selected

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30958]
## Description
Does not announce "selected" on the focus of ProgressBarAndroid Component.

This issue is covered in-depth by parent issue #30842, please check there for more details.

## React Native version:
0.63

**Comment by fabriziobertoglio1987** [comment]
ProgressBarAndroid is deprecated https://reactnative.dev/docs/progressbarandroid https://github.com/facebook/react-native/issues/23313 ?
https://github.com/react-native-progress-view/progress-bar-android 
Thanks :peace_symbol: :pray:

**Comment by hardikshah197** [comment]
I'll take up this issue, and working on it, for resolving out this one
Thanks : )

**Comment by amarlette** [comment]
Thank you @hardikshah197 for taking on this issue! I have created this issue in the react-native-progress-view/progress-bar-android repo. That repo is the most updated version of the ProgressBarAndroid Component  https://github.com/react-native-progress-view/progress-bar-android/issues/64 

Closing this since it is deprecated and is being worked on in another repo.
